packages/inventory/src/redux/entities.js

```diff
@@ -36,6 +37,14 @@ export const defaultColumns = [
         title: 'Name',
         renderFunc: TitleColumn
     },
+    {
+        key: 'system_profile',
+        title: 'OS release',
+        // eslint-disable-next-line react/display-name
+        renderFunc: (systemProfile) => systemProfile.os_release || 'Not available',
+        props: { isStatic: true },
```

Comment:
Looking at the screenshot the width of the cell is not that fancy. I'd add the width prop here anyway.
```suggestion
        props: { width: 25, isStatic: true },
```